includes/fbproduct.php

```diff
@@ -199,8 +199,7 @@ public function get_all_image_urls() {
 			$image_urls = array_filter( array_unique( $image_urls ) );
 
 			if ( empty( $image_urls ) ) {
-				// TODO: replace or remove this placeholder - placeholdit.imgix.net is no longer available {WV 2020-01-21}
-				$image_urls[] = sprintf( 'https://placeholdit.imgix.net/~text?txtsize=33&name=%s&w=530&h=530', rawurlencode( strip_tags( $this->woo_product->get_title() ) ) );
+				$image_urls[] = plugin_dir_url( dirname( __FILE__ ) ) . 'assets/images/woocommerce-placeholder.png';
```

Comment:
```suggestion
				$image_urls[] = facebook_for_woocommerce()->get_plugin_url() . '/assets/images/woocommerce-placeholder.png';
```
I feel like this is simpler...